bitmark-references/rpc-communication/rpc-communication.md

```diff
@@ -0,0 +1,129 @@
+# RPC
+
+## Introduction
```

Comment:
To me this section is quite redundant. We can presume audiences know RPC before, or cite to a public wiki page for definition of RPC.

Reply:
Instead, I think we can introduce what kind of RPC protocol that bitmarkd is using.

Reply:
OK, remove most of RPC introduction